content/beginner/160_advanced-networking/secondary_cidr/eniconfig_crd.md

```diff
@@ -47,45 +48,95 @@ aws ec2 describe-subnets  --filters "Name=cidr-block,Values=100.64.*" --query 'S
 |  100.64.0.0/19  |  subnet-04f960ffc8be6865c  |  us-east-2b |
 +-----------------+----------------------------+-------------+
 {{< /output >}}
+
+Ensure new nodes are up and listed with 'Ready' status. 
+```
+kubectl get nodes  # Make sure new nodes are listed with 'Ready' status
+```
+{{ output }}
+NAME                                           STATUS   ROLES    AGE   VERSION
+ip-192-168-9-228.us-east-2.compute.internal     Ready    <none>   90m   v1.17.12-eks-7684af
```

Comment:
These nodes should be 1.20 to match the expected version upgrade.

Reply:
Applied Fixed 
https://github.com/aws-samples/eks-workshop/pull/1178/commits/8b26c5501b527ddff27b4a01ca51c7c4bf74e00b